3 reports.sql

```diff
@@ -0,0 +1,37 @@
+/*
+1)Inorder to see which flavor is most popular, can you show me how long has each flavor been 
+sitting on the shelf before it was sold (per chocolate flavor and chocolateType)? overall which flavor
+was on the shelf the longest? which the shortest?
+*/
+
```

Comment:
this does not answer the last 2 parts of the question

Reply:
Do you want more than one select statement? I'm not sure how I could do it in one...
(I slacked you about this when you were out ;) )